### ISSUE TITLE
Feature: Add config for increasing flexible-block-placement UI size

### ISSUE THREAD

**Issue by Andrews54757** [maruohon/tweakeroo#50]
It is really difficult sometimes to correctly place blocks using flexible block placement if one is using a smaller screen -- the edge boxes are way too small. An option to increase the size of those boxes would be very helpful.

**Comment by maruohon** [comment]
Yep I was thinking about this myself a little while ago. I now added it to my TODO list since it didn't seem to be there yet.
